torch/testing/_internal/distributed/nn/api/remote_module_test.py

```diff
@@ -257,8 +257,7 @@ def test_invalid_devices(self):
 
         with self.assertRaisesRegex(
             RuntimeError,
-            r"Expected one of cpu, cuda, xpu, mkldnn, opengl, opencl, ideep, hip, msnpu, mlc, xla, vulkan"
-            " device type at start of device string",
+            r"Expected one of .+ device type at start of device string",
```

Comment:
Is this related?

Unless this is somehow regex-filling-in all possible device types and I don't see how 😛

Reply:
Yeah, I changed the list of valid device types, so this test is wrong. I just got rid of the list match entirely and replaced it with a wildcard.